frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -23,11 +23,11 @@ import { Flavor } from '../utils/constants/wizard';
 import { resourceHorizontalTab } from '../../views/uiResource.view';
 
 export class Wizard {
-  async openWizard(kind: K8sKind) {
-    if (!(await resourceTitle.isPresent()) || (await resourceTitle.getText()) !== kind) {
+  async openWizard(model: K8sKind) {
+    if (!(await resourceTitle.isPresent()) || (await resourceTitle.getText()) !== model) {
```

Comment:
isn't this always going to return true ? ` await resourceTitle.getText()) !== model`

Reply:
Indeed, thanks, I noticed that I also didn't update the resourceTitle (the Virtualization title has different attributes than on other pages)

Reply:
+1